src/components/index.ts

```diff
@@ -1,13 +1,13 @@
 export { default as ArcLogo } from './ArcLogo';
 export { default as ErrorBoundary } from './ErrorBoundary';
+export { default as EventEmitter } from './EventEmitter';
```

Comment:
alphabetized?

Reply:
bad habit and a simple click of a button in sublime :p